resources/kernel/make_busybox.sh

```diff
@@ -115,7 +115,7 @@ mount -t sysfs none /sys
 EOF
 
     if [ -z "$halt" ]; then
-        echo "exec /bin/sh" >>init
+        echo "setsid sh -c 'exec sh </dev/ttyS0 >/dev/ttyS0 2>&1'" >>init
```

Comment:
I think this needs the full path to `sh`, otherwise this happens:
```
Hello, world, from the rust-vmm reference VMM!
setsid: can't execute 'sh': No such file or directory
[    0.737487] Kernel panic - not syncing: Attempted to kill init! exitcode=0x00007f00
```

```suggestion
        echo "setsid /bin/sh -c 'exec /bin/sh </dev/ttyS0 >/dev/ttyS0 2>&1'" >>init
```

Reply:
CI should fail because of that, but it doesn't actually rebuild the images because of this: https://github.com/rust-vmm/vmm-reference/blob/3f8a44d5ddb1824804a67abb947e0de66d47cac3/resources/kernel/make_busybox.sh#L187
We should figure out a way to force a kernel rebuild when necessary, without making a CI run last for hours.

Reply:
I removed this step, if we don't change something in the script that is related to kernel, the rebuild should not be retriggered.

Reply:
I still see the kernel panic because of `/bin/sh` on a fresh build.

Reply:
It is rather strange as this happens neither in the CI nor on my machine :(. Anyway, I updated with the full path, thanks!